SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/TimeToLive.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.AppCenter.Data
+{
+    /// <summary>
+    /// Constants for document time to live on device.
+    /// </summary>
+    public static class TimeToLive
+    {
+        /// <summary>
+        /// Cache does not expire.
+        /// </summary>
+        public static readonly TimeSpan Infinite = TimeSpan.FromSeconds(-1);
+
+        /// <summary>
+        /// Do not cache documents.
+        /// </summary>
+        public static readonly TimeSpan NoCache = TimeSpan.Zero;
+
+        /// <summary>
+        /// Default caching value of one day (in seconds).
```

Comment:
```suggestion
        /// Default caching value of one day.
```
TimeSpan is not based on seconds or any unit.